pkg/templates/git.go

```diff
@@ -6,6 +6,9 @@ import (
 	"os"
 	"path/filepath"
 
+	settingbus "github.com/seal-io/walrus/pkg/bus/setting"
+	"github.com/seal-io/walrus/pkg/settings"
+
```

Comment:
fix `goimport` please, and move the new imports to the bottom.

Reply:
updated